test/unit/presenters/local_transaction_presenter_test.rb

```diff
@@ -16,4 +24,37 @@ def subject(content_item)
   test "#need_to_know" do
     assert_equal "foo", subject(details: { need_to_know: "foo" }).need_to_know
   end
+
+  test "#scotland_availability" do
+    assert_equal "devolved_administration_service", subject(@content_item_with_devolved_administration_availability, "Scotland").scotland_availability["type"]
+    assert_equal "https://www.gov.scot/stray-dog", subject(@content_item_with_devolved_administration_availability, "Scotland").scotland_availability["alternative_url"]
+  end
+
+  test "#wales_availability" do
+    assert_equal "unavailable", subject(@content_item_with_devolved_administration_availability, "Wales").wales_availability["type"]
+    assert_equal nil, subject(@content_item_with_devolved_administration_availability, "Wales").wales_availability["alternative_url"]
+  end
+
+  test "#northern_ireland_availability" do
+    assert_equal "devolved_administration_service", subject(@content_item_with_devolved_administration_availability, "Northern Ireland").northern_ireland_availability["type"]
+    assert_equal "https://www.nidirect.gov.uk/stray-dog", subject(@content_item_with_devolved_administration_availability, "Northern Ireland").northern_ireland_availability["alternative_url"]
+  end
+
+  test "#unavailable?" do
```

Comment:
I think this should be broken up into tests for the different scenarios:

```
test "#unvailable? returns true for a devolved administration marked as unavailable"
test "#unvailable? returns false for a devolved administration marked as devolved_administration_service"
test "#unvailable? returns false for a non devolved administration"
```

I think these can also be self contained which should make them a little easier to understand.

Reply:
👍

Reply:
👍